app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -39,13 +41,31 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
               label="Title"
               name="title"
```

Comment:
One thing about using `validationSchema` and `initialValues` in formik forms. Please do not place these inside a constants file that is globally scoped. 
Please move these constants to a file in the scope of `Notes` directory.